tests/unittests/test_core/test_archiver.py

```diff
@@ -23,7 +25,8 @@
 HEAD_NAMES = [head_name for head_name in ImplementedHeads.__members__]
 
 
-class TestArchiver:
+@pytest.mark.skip()
```

Comment:
Why do we skip this test?

Reply:
The NN Archive in the latest dev of luxonis-ml is not compatible with the tests for now (we need to figure out the yolov8 keypoint outputs in the archive)